spec/models/defence_request_transitions/acknowledge_spec.rb

```diff
@@ -1,51 +1,71 @@
-require_relative "../../../app/models/defence_request_transitions/acknowledge"
-require "securerandom"
-require "transitions"
+require "rails_helper"
 
-RSpec.describe DefenceRequestTransitions::Acknowledge, "#complete" do
-  it "transitions the defence request to acknowledge state" do
-    defence_request = spy(:defence_request)
-    user = spy(:user, uid: SecureRandom.uuid)
+RSpec.describe DefenceRequestTransitions::Acknowledge, "#complete", :mock_auth_api do
+  let(:defence_request) { create(:defence_request, :duty_solicitor, :queued, dscc_number: nil ) }
+  let(:user) { spy(:user, uid: SecureRandom.uuid) }
```

Comment:
Any reason for making user a spy? We are not running assertions against the object and, since we are depending on Rails, it seems we might as well use a full fledged object here.